### PR TITLE
added project doc for failure type classification in CI testgrid

### PR DESCRIPTION
**Pull request by SankBad**
## Related Issues and Dependencies

This issue address #25 

## This introduces a breaking change

- [ ] Yes
- [ ] No

## This Pull Request implements

This PR will add the project doc for "Failure type classification with the TestGrid data"

## Description

Currently, we added a basic method to classify/detect different types of failures in test grid data. We will update this doc as we include more methods.

### CODE REVIEW

**Comment by SankBad** [comment]
Hi @MichaelClifford , 

The last PR was got closed after rebase. Hence, created a new PR.

File: docs/publish/failure-type-classification-with-the-testgrid-data-project-doc.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## Weekly Meeting Updates
+
+*Keep track of ongoing meetings below.*
+
+### Year-Month-Day
+
+* Item 1
+
+* Item 2
```
Comment (MichaelClifford) [inline docs/publish/failure-type-classification-with-the-testgrid-data-project-doc.md:75]: you can remove this section

File: docs/publish/failure-type-classification-with-the-testgrid-data-project-doc.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+4. Results hosted for SME to review
+
+### C. Provide a solution in terms of human actions to confirm if the task is within the scope of automation through AI.
+
+Without AI and automation tooling, SME will need to go to TestGrid data of a particular release and look at the failures. An SME will determine if that failure is following any of the patterns that we have discussed in earlier sections. Based on detected patterns, an SME tries to determine the reason behind failure.
+
+### D. Outline a path to operationalization.
+
+Once we have notebooks ready, we will build a Notebook-based Pipeline using Elyra. The results will be stored in S3. We can then use Superset as our dashboard and visualization tool, which SME/developers can access and give feedback. Once SMEs are satisfied with the result, we can create PR to integrate our work directly into the TestGrid platform.
```
Comment (MichaelClifford) [inline docs/publish/failure-type-classification-with-the-testgrid-data-project-doc.md:49]:
```suggestion
Once we have notebooks ready, we will build a Notebook-based Pipeline using Elyra. The results will be stored in S3. We can then use Superset as our dashboard and visualization tool, which SME/developers can access and give feedback. If the tool is deemed useful, we could also look into integrating it with the existing TestGrid project. 
```

**Review by MichaelClifford — CHANGES_REQUESTED** [review]

File: docs/publish/failure-type-classification-with-the-testgrid-data-project-doc.md (hunk 1 above)
Comment (SankBad) [reply docs/publish/failure-type-classification-with-the-testgrid-data-project-doc.md:75]: Hi @MichaelClifford , 
I am planning to remove this section except the dataset part. 
what do you say?

**Comment by SankBad** [comment]
Hi @MichaelClifford , 

Thanks for the feedback. Incorporated the suggestions.

**Review by MichaelClifford — APPROVED** [review]
lgtm

**Pull request merged**